ConwaysGameOfLife.Tests/LocationShould.cs

```diff
@@ -1,41 +1,13 @@
 using System;
+using System.Collections.Generic;
 using Xunit;
 
 namespace ConwaysGameOfLife.Tests
 {
     public class LocationShould
     {
-        [Fact]
-        public void WhenXAndYAreSame_BeEqual()
-        {
-            var x = 0;
-            var y = 0;
-            var location = new Location(x, y);
-            var one = location;
-            var two = new Location(0, 0);
-
-            var result = (one.Equals(two));
-
-            Assert.True(result);
-        }
-
-        [Fact]
-        public void WhenXAndYAreSame_BeEqualOperator()
-        {
-            var x = 0;
-            var y = 0;
-            var location = new Location(x, y);
-            var one = location;
-            var two = new Location(0, 0);
-
-            var result = (one == two);
-
-            Assert.True(result);
-        }
-
         [Theory]
         [InlineData(0, 0, 0, 0, true)]
-        [InlineData(0, 0, 1, 0, false)]
         public void WhenTwoLocationsInstantiated_BeEqual(int x1, int y1, int x2, int y2, bool expected)

```

Comment:
No need to say Instantiated as we're not really testing the constructor for a class you've written

```suggestion
        public void GivenTwoLocationsOfSameCoordinates_BeEqual(int x1, int y1, int x2, int y2)
```

Reply:
Same goes for the `WhenTwoLocationsInstantiated_NotBeEqual `test  i.e. could be renamed to `GivenTwoLocationsOfDifferingCoordinates_NotBeEqual`